projects/playground/e2e/integration/playground.spec.ts

```diff
@@ -33,7 +33,7 @@ describe('Smoke tests', () => {
 
     loginToAuth0();
 
-    cy.get('[data-cy=logout-local]').should('not.be.checked');
+    cy.get('[data-cy=logout-localOnly]').should('not.be.checked');
```

Comment:
So this is interesting, this test started to fail but there is no element with `data-cy=logout-local` ...

Reply:
Do we know why this has only just started failing?

Reply:
I can't find anything specific in the Cypress release notes (https://docs.cypress.io/guides/references/changelog.html#5-6-0). But I think it should have never succeeded.